### ISSUE TITLE
fakeredis.aioredis isn't compatible with aioredis v2+

### ISSUE THREAD

**Issue by egnor** [jamesls/fakeredis#299]
Info on aioredis v2: https://aioredis.readthedocs.io/en/latest/migration/

Attempting to import fakeredis.aioredis fails right away since aioredis.ConnectionsPool no longer exists (and I suspect that's just the tip of the iceberg, since basically everything has changed).

If nothing else it might be worth posting a warning on the site about the incompatibility.

**Comment by bmerry** [comment]
Thanks for the heads-up. I didn't realise that aioredis 2.x had been released. It looks like it's really a completely new library and the aioredis support in fakeredis will need to be completely rewritten. For now I'll add a `<2` to the dependency and add a note to the README, as I'm not sure when I'll have time for that rewrite.

**Comment by egnor** [comment]
(I'm not sure it's *released* released (out of beta) yet, but it's
certainly available...)

**Comment by bmerry** [comment]
You're right, the changelog lists it as released in March, but it's not yet on PyPI.

**Comment by bmerry** [comment]
I've pushed some experimental work onto a `aioredis2` branch. It's not production-ready (e.g. documentation still needs to be updated), but give it a try and let me know if it works for you. It works similarly to the redis-py support i.e. create a `fakeredis.aioredis.FakeRedis`.

**Comment by egnor** [comment]
Ooh that's nice of you! I ended up using redislite for my purposes (so I could simply use aioredis2 as-is, talking to a new URL)... of course there are tradeoffs there as you're aware... but if I end up circling back (or someone else sees this thread!) will definitely give it (or whatever you've got going) a try.
